### ISSUE TITLE
Option to disable emojis

### ISSUE THREAD

**Issue by pawamoy** [pykong/copier#189]
There are still of lot of systems/terminal emulators/fonts that do not support emojis. Instead, ugly "unknown" squares will be rendered, and I know it can be quite annoying for users, especially when they don't know it's supposed to be emojis. They might think the output is broken or something.

Therefore I'd suggest adding an option to replace emojis with pieces of text. I would be great to detect if emojis are supported, though I don't think it's possible, or at least not simple to do.

**Comment by Yajo** [comment]
I'm not sure I want to fix this. You're gonna have problems with more things like helm, pre-commit... Displaying emojis in a terminal is quite common these days, and most times all you need to do is add the required font package.

Do you have any practical use cases?

This might be worth reading: https://unix.stackexchange.com/q/284990/50281

**Comment by pawamoy** [comment]
> You're gonna have problems with more things like helm, pre-commit...

True... might not be worth. Just like for `--color=always` options... If I had to open issues on every program that doesn't have it, I'd spend years on it :sweat_smile: 

It's just that I never managed to get a font that correctly renders emoji on Debian 9 (I changed OS since), and I think it would be a pain on Windows as well. Especially if you're in a corporate environment and don't have the privileges to install fonts.

But anyway, let's not waste energy on this!
